test/mocks.js

```diff
@@ -126,6 +126,9 @@ function mockDB (data, errors) {
         wrapWrapKb: crypto.randomBytes(32)
       })
     }),
+    forgotPasswordVerified: sinon.spy(function () {
```

Comment:
You don't actually need to define the body of this function fwiw. `mockObject` provides a default implementation that returns an empty promise, the ones listed here are only to over-ride that with something different.